src/index.tsx

```diff
@@ -67,21 +68,30 @@ export default function Command() {
                 <ActionPanel>
                   <Action.CopyToClipboard
                     content={employee.email}
-                    title="Kopier E-Post Til Utklippstavle"
+                    title="Kopier E-Post"
                     shortcut={{ modifiers: ["cmd"], key: "e" }}
                   />
                   <Action.CopyToClipboard
                     content={employee.name}
-                    title="Kopier Navn Til Utklippstavle"
+                    title="Kopier Navn"
                     shortcut={{ modifiers: ["cmd"], key: "n" }}
                   />
                   {employee.telephone != null && (
                     <Action.CopyToClipboard
                       content={employee.telephone || ""}
-                      title="Kopier Telefonnummer Til Utklippstavle"
+                      title="Kopier Telefonnummer"
```

Comment:
```suggestion
                      title="Kopier telefonnummer"
```

Reply:
Gir eslint-feil, men kan vel ignoreres.

Reply:
For å sitere en bitter mann fra en annen tråd (#4 )

> hmmmmmmmmmmmmmmmmmm. scheeeptisk. Anglifisert regel

Reply:
😂